tag_branch_consistency

```diff
@@ -13,11 +13,13 @@ if [ ! -f ${DIR}/BRANCH ]; then
 fi
 
 TAG=$(cat "${DIR}"/TAG)
-BRANCH=$(cat "${DIR}"/BRANCH)
+BRANCH=$(cat ${DIR}/BRANCH | xargs)
```

Comment:
This command works, but using `xargs` here feels a bit implicit.
How about something like this:

```bash
BRANCH=$(tr -d '\n' < "${DIR}/BRANCH")
```